oidc/client.go

```diff
@@ -66,11 +66,16 @@ type ClientConfig struct {
 }
 
 func NewClient(cfg ClientConfig) (*Client, error) {
+	ru, err := phttp.ParseNonEmptyURL(cfg.RedirectURL)
```

Comment:
I see... that makes sense. Feel free to remove this validation if it helps your case.